### PR TITLE
VSCode Configs Changes

### PR DESCRIPTION
**Pull request by Lucien950**
### Changelist 

- tasks use cmake command instead (idk how to make this work on windows too 😭)
- reorganize launch configs and remove thruna launch configs

### Testing Done

None, checking in vscode.

### Resolved Issues

None

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [x] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [x] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
       "symbolFiles": [
         // This order matters, or RTT won't work! (app has to be last)
         "build_fw_deploy/firmware/quadruna/RSM/quadruna_RSM_boot.elf",
-        "build_fw_deploy/firmware/quadruna/RSM/quadruna_RSM_app.elf",
+        "build_fw_deploy/firmware/quadruna/RSM/quadruna_RSM_app.elf"
```
Comment (Lucien950) [inline .vscode/launch.json:101]:
```suggestion
        "build_fw_deploy/firmware/quadruna/RSM/quadruna_RSM_app.elf",
```

File: .vscode/launch.json [hunk 2]
```diff
@@ -119,10 +119,10 @@
             "type": "console"
           }
         ]
-      },
+      }
```
Comment (Lucien950) [inline .vscode/launch.json:122]:
```suggestion
      },
```

File: .vscode/launch.json [hunk 3]
```diff
@@ -181,6 +181,7 @@
       "cwd": "${workspaceRoot}",
       "executable": "build_fw_deploy/firmware/dev/h7dev/h7dev.hex",
       "symbolFiles": [
+        // This order matters, or RTT won't work! (app has to be last)
```
Comment (Lucien950) [inline .vscode/launch.json:184]:
```suggestion
```

File: .vscode/launch.json [hunk 4]
```diff
@@ -122,7 +122,7 @@
       },
     },
     {
-      "name": "Debug Embedded: CRIT (Quadruna)",
+      "name": "Debug Embedded: RSM (Quadruna)",
```
Comment (Lucien950) [inline .vscode/launch.json:125]:
```suggestion
      "name": "Debug Embedded: CRIT (Quadruna)",
```

File: .vscode/launch.json [hunk 5]
```diff
@@ -144,7 +144,7 @@
         "clearSearch": false, // Required for OpenOCD
         "decoders": [
           {
-            "label": "Debug logs: CRIT",
+            "label": "Debug logs: RSM",
```
Comment (Lucien950) [inline .vscode/launch.json:147]:
```suggestion
            "label": "Debug logs: CRIT",
```

**Review by gtaharaedmonds — APPROVED** [review]
yeee

**Pull request merged**